caluma_client/tests/test_parser.py

```diff
@@ -0,0 +1,13 @@
+import json
+from pathlib import Path
+
+from caluma_client.models import Document
+from caluma_client.parser import parse_document
+
+
+def test_parse_full_document():
+    response_file = Path(__file__).parent / "files/full_form_response.json"
+    data = json.load(response_file.open())
+
+    raw = parse_document(data, nesting="data.allDocuments.edges.0.node")
+    Document(raw)
```

Comment:
Would it be possible to serialize the parsing result somehow? (and use snapshot testing on it?)

Reply:
So you suggest we do a kind of round-trip testing (JSON -> models -> JSON) and verify using snapshots? It should be possible as I plan to implement visitors anyway, and the generic visitor could just print `self.raw` for each of the models.

Reply:
Sounds good! I'd do it only if it's simple to implement and if it helps while development.